Formula/httprs.rb

```diff
@@ -0,0 +1,18 @@
+class Httprs < Formula
+  desc "Simple and configurable command-line HTTP server"
+  homepage "https://github.com/http-server-rs"
+  url "https://github.com/http-server-rs/http-server/releases/latest/download/http-server_macos-latest"
```

Comment:
- we need to build from source

Reply:
you can find some inspirations from the other rust formulae.

Reply:
I think this will help? https://github.com/Homebrew/homebrew-core/pull/132209/commits/5165eb24c7046b0a41a4ee886cf479c552be536b